misc/test_site/mac_addrs/9a02571e8f01/module_config.json

```diff
@@ -1,15 +1,44 @@
 {
+  "device_datasheets_url": "*** Device Datasheets URL ***",
+  "device_description": "*** Device Description ***",
+  "device_image": "*** Device Image URL ***",
   "device_info": {
+    "guid": "*** GUID ***",
+    "hostname": "*** Network Hostname ***",
+    "make": "*** Make ***",
+    "model": "*** Model ***",
+    "name": "*** Name ***",
+    "serial": "*** Serial ***",
+    "type": "*** Type ***",
+    "firmware_version": "*** Firmware Version ***"
   },
+  "device_manuals_url": "*** Device Manuals URL ***",
   "modules": {
+    "bacnet": {
+      "enabled": true
+    },
     "brute": {
       "enabled": true
     },
+    "mudgee": {
```

Comment:
why is this added?

Reply:
It was a mistake from copy-pasting.
Now fixed.